spyder/app/tests/test_mainwindow.py

```diff
@@ -90,6 +89,7 @@
 from spyder.utils.clipboard_helper import CLIPBOARD_HELPER
 from spyder.utils.programs import find_program
 from spyder.widgets.dock import DockTitleBar
+from spyder_kernels.utils.pythonenv import is_conda_env
```

Comment:
This is a third-party import, so please move it to the section above this one (i.e. around line **43**).